crates/store/re_remote_store_types/src/lib.rs

```diff
@@ -0,0 +1,339 @@
+                .filtered_pov
+                .map(|fp| {
+                    re_dataframe::external::re_chunk_store::ComponentColumnSelector::try_from(fp)
+                })
+                .transpose()?;
+
+            Ok(Self {
+                view_contents: value.view_contents.map(|vc| vc.into()),
+                filtered_index,
+                filtered_index_range: value
+                    .filtered_index_range
+                    .map(|ir| ir.try_into())
+                    .transpose()?,
+                filtered_index_values: value
+                    .filtered_index_values
+                    .map(|iv| iv.time_points.into_iter().map(|v| v.into()).collect()),
+                using_index_values: value
+                    .using_index_values
+                    .map(|uiv| uiv.time_points.into_iter().map(|v| v.into()).collect()),
+                filtered_point_of_view,
+                sparse_fill_strategy:
+                    re_dataframe::external::re_chunk_store::SparseFillStrategy::default(), // TODO(zehiko) implement support for sparse fill strategy
+                selection,
+            })
+        }
+    }
+
+    impl From<ViewContents> for re_dataframe::external::re_chunk_store::ViewContentsSelector {
+        fn from(value: ViewContents) -> Self {
+            value
+                .contents
+                .into_iter()
+                .map(|part| {
+                    // TODO(zehiko) option unwrap
+                    let entity_path = Into::<re_log_types::EntityPath>::into(part.path.unwrap());
+                    let column_selector = part.components.map(|cs| {
+                        cs.components
+                            .into_iter()
+                            .map(|c| re_dataframe::external::re_chunk::ComponentName::new(&c.name))
+                            .collect::<BTreeSet<_>>()
+                    });
+                    (entity_path, column_selector)
+                })
+                .collect::<Self>()
+        }
+    }
+
+    impl From<EntityPath> for re_log_types::EntityPath {
+        fn from(value: EntityPath) -> Self {
+            Self::from(value.path)
+        }
+    }
+
+    impl TryFrom<IndexColumnSelector> for re_log_types::Timeline {
+        type Error = TypeConversionError;
+
+        fn try_from(value: IndexColumnSelector) -> Result<Self, Self::Error> {
+            let timeline_name = value
+                .timeline
+                .ok_or(TypeConversionError::MissingField("timeline"))?
+                .name;
+
+            #[allow(clippy::match_same_arms)]
+            let timeline = match timeline_name.as_str() {
+                "log_time" => Self::new_temporal(timeline_name),
+                "log_tick" => Self::new_sequence(timeline_name),
+                "frame" => Self::new_sequence(timeline_name),
+                "frame_nr" => Self::new_sequence(timeline_name),
+                _ => Self::new_temporal(timeline_name),
+            };
```

Comment:
Oof, that's a huge issue.

This needs a `TODO(cmc): QueryExpression::filtered_index gotta be a selector`.